.github/workflows/pr-ci.yml

```diff
@@ -85,14 +118,38 @@ jobs:
       checkout-repo: ${{ needs.checkout-branch.outputs.PR_repo }}
       checkout-ref: ${{ needs.checkout-branch.outputs.PR_head_ref  }}
       skip-itests: ${{ matrix.arch != 'amd64' }}
+  
+  check-test-results:
+    runs-on: ubuntu-latest
+    needs: [build-and-test]
+    steps:
+      - name: Check Test Results
+        id: check-results
+        run: |
+          if [ ... ]; then
```

Comment:
Does this if-condition still need to be filled in?

Reply:
Yes, I kind of have difficulties in figuring out this part.. The option that makes sense is to read test results in a file and grep if there is a test that failed. Any help with this part will be highly appreciated.. on may be another logic to log the test pass or failed???

Reply:
The exit status of the `mvn` command will suffice. If `mvn test` exits with the normal 0 status then all tests passed, if it exits with `1` then at least one unit test failed. Likewise for `mvn verify` and integration tests.

The `repeated-integration-tests.bash` uses this exit status to count passes/failures, for example, and this script also uses the number of failures as its exit status, so if the exit status is 0 then all tests passed, otherwise there was a failure.

Reply:
YES. Thank you.